auto_submit/lib/service/validation_service.dart

```diff
@@ -205,6 +208,7 @@ class ValidationService {
 
       log.info(message);
     } else {
+      log.info('Pull Request #$prNumber was merged successfully!');
       log.info('Attempting to insert a pull request record into the database for $prNumber');
```

Comment:
Nit: add the slug to make it easy to track which repo the PR is from.

Reply:
Done.